metabase/metabase
# Add metric icons in the notebook editor

Closes https://github.com/metabase/metabase/issues/42356

Adds metric icons in Data and Join steps if the card is a metric.

How to verify:
- New -> Question -> Select metric
- Join a metric
- See that there are new icons
- For regular tables/questions should should be no icons 

Please note that `Orders` is displayed incorrectly here because of a MBQL lib bug https://github.com/metabase/metabase/issues/42390.

<img width="1297" alt="Screenshot 2024-05-08 at 10 11 38" src="https://github.com/metabase/metabase/assets/8542534/94c9d82a-920a-4bf3-a1bc-9a2ff8bcfe6f">
